drivers/gles2/shaders/tonemap.glsl

```diff
@@ -1,19 +1,18 @@
 /* clang-format off */
 [vertex]
 
-layout(location = 0) in highp vec4 vertex_attrib;
 /* clang-format on */
-layout(location = 4) in vec2 uv_in;
 
-out vec2 uv_interp;
+#include "stdlib.glsl"
+attribute vec2 vertex_attrib; // attrib:0
```

Comment:
`clang-format` is a bit confused by our `[vertex]` and `[fragment]` tags, so the `/* clang-format on */` coming after the first attribute definition was not a mistake and should be kept as is.

Otherwise we get this: https://travis-ci.org/godotengine/godot/jobs/579296139